doc/installing/README.md

```diff
@@ -1,46 +1,88 @@
-# Installing the OpenQuake Engine: overview
+# Installing the OpenQuake Engine
 
-The OpenQuake Engine can be installed in several different ways. The
-easiest and recommended way is to use the [universal installation
-script](universal.md) which works for any platform and for any version
-of the engine.
+The OpenQuake Engine can be installed in several different ways.
+The engine runs on Linux, macOS and Windows; on laptops, workstations, standalone servers and multi-node clusters.
 
-If you are on Windows and you do not need to interact with any other Python software you may also use the **[self-installable binary distribution for Windows 10](windows.md)** (available only for LTS versions). Administrator level access may be required.
-If you are on Linux and you do not need to interact with any other Python software you may also use the **binary packages for [Ubuntu](ubuntu.md)** or **[RedHat (and clones)](rhel.md)** (available only for LTS versions) that require root access.
+> **Warning**:
+> If you already have an engine installation, before installing the new version you must [uninstall](universal.md#uninstalling-the-engine) the old one.
```

Comment:
btw we also have this page: https://github.com/gem/oq-engine/tree/doc_installation/doc/upgrading

Reply:
Removed since obsolete